media_utils/video_puppeteer.py

```diff
@@ -56,12 +39,13 @@ def __init__(self, marionette, url, video_selector='video'):
                                                           'Using first.')
             if len(videos_found) > 0:
                 self.video = videos_found[0]
-                self.marionette.execute_script("log('video element obtained');")
-
-    def get_debug_lines(self):
-        with self.marionette.using_context('chrome'):
-            debug_lines = self.marionette.execute_script(self._debug_script)
-        return debug_lines
+                self.marionette.execute_script(
+                    "log('video element obtained');")
+                self.start_time = self.current_time
+                self.start_wall_time = clock()
+                self.initial_lag = 0
+                self.set_duration = duration
```

Comment:
I think this would be easier to understand if the `duration` argument to `__init__` were named `set_duration`,  otherwise it clashes (at least, conceptually) with `self.duration`. 

(So, this line would change to `self.set_duration = set_duration`.)